.travis.yml

```diff
@@ -1,11 +1,24 @@
+sudo: required
 language: python
 python: 3.6
-env:
-  - PIPENV_IGNORE_VIRTUALENVS=1
+services:
+  - docker
+branches:
+  only:
+    - master
+    - job-dsl
 install:
   - pip install -r pipenv.txt
-script:
   - pipenv install --dev
-  - pipenv run flake8
+jobs:
+  include:
+    - stage: Linting
+      script:
+        - pipenv check
+        - pipenv install flake8
```

Comment:
Note that this installs the latest version of flake8 rather than the pinned version from `Pipfile.lock` that would be installed using `pipenv install --dev`.

Reply:
Thx; fixing